Counter/src/app/intro/intro.component.html

```diff
@@ -0,0 +1,24 @@
+<h2>A Pokémon Shiny Counter</h2>
+<div class="row intro-container">
+    <div class="col-lg-6 order-lg-1">
+        <img class="img-fluid my-3" src="https://archives.bulbagarden.net/media/upload/4/44/Pok%C3%A9mon_Gallery_Encounter_with_Shining.png">
+    </div>
+    <div class="col-lg-6 order-lg-2 intro">
+        <p>This website app is designed to store your current shiny hunts in one place! You can store 
+            multiple hunts at the same time. Each Pokémon will be displayed with an image and the 
+            current amount of encounters. When you have finally gotten that shiny, you can add it 
+            to the hall of fame where all your successful hunts will be stored along with the amount 
+            of encounters it took. Happy hunting!
+        </p>
+        <div class="bottom-buttons">
+            <button class="btn btn-primary left-button" routerLink="/newaccount" 
+            *ngIf="!(isLoggedIn | async)">Get started here!</button>
```

Comment:
Every time you pipe async that isLoggedIn it's going to create a subscriber for that. In this case we're adding a lot of subscribers when we could probably just add one with an `ng-container` or at the div level. You could use [ngif with else syntax](https://angular.io/api/common/NgIf#description) to accomplish this.

Reply:
I have changed the buttons to use an if-else syntax.